client/src/assets/stylesheets/partials/_base.scss

```diff
@@ -7,6 +7,8 @@ $baseBg: #f9f9f9;
 
 $progress-color: #2bc252;
 
+$incorrect-char-color: #F44336!important;
```

Comment:
We should be able to accomplish this without using `!important`. Avoid using `!important` at all costs, there is always a better way.